### ISSUE TITLE
Persistent modal bar

### ISSUE THREAD

**Issue by tom-christie** [ropensci/plotly#319]
If I use the basic example:

    library(plotly)
    set.seed(100)
    d <- diamonds[sample(nrow(diamonds), 1000), ]
    p <- ggplot(data = d, aes(x = carat, y = price)) + 
        geom_point(aes(text = paste("Clarity:", clarity))) +
        geom_smooth(aes(colour = cut, fill = cut)) + facet_wrap(~ cut)
    (gg <- ggplotly(p))

and then configure:

    config(gg,showLink=F, sendData=F, displaylogo=F, displayModeBar = F)

the modal bar doesn't appear. However, if you click and drag to zoom in, the modal bar reappears (WITH the link to "Save and Edit Plot in Cloud").  Presumably this is undesired behavior...it certainly is undesired for me, as I am trying keep people from uploading the private data I'm hosting up to Plotly's cloud with the click of a button.

Edit: interestingly, the modal disappears if I interact with the plot again, and then reappears if I interact a third time, and so on.

**Comment by cpsievert** [comment]
Hmm, I can't seem to replicate this behavior. Could you provide the output of `devtools::session_info()` as well?

**Comment by tom-christie** [comment]
Happy to:

    Session info -------------------------------------------------------------------------
    setting  value                       
    version  R version 3.2.2 (2015-08-14)
    system   x86_64, darwin15.0.0        
    ui       RStudio (0.99.489)          
    language (EN)                        
    collate  en_US.UTF-8                 
    tz       America/Chicago             
    date     2015-12-01                  
  
    Packages -----------------------------------------------------------------------------
    package     * version date       source                          
    base64enc     0.1-3   2015-07-28 CRAN (R 3.2.0)                  
    codetools     0.2-14  2015-07-15 CRAN (R 3.2.2)                  
    colorspace    1.2-6   2015-03-11 CRAN (R 3.2.2)                  
    devtools      1.9.1   2015-09-11 CRAN (R 3.2.2)                  
    digest        0.6.8   2014-12-31 CRAN (R 3.2.2)                  
    ggplot2     * 1.0.1   2015-03-17 CRAN (R 3.2.2)                  
    gridExtra     2.0.0   2015-07-14 CRAN (R 3.2.0)                  
    gtable        0.1.2   2012-12-05 CRAN (R 3.2.2)                  
    htmltools     0.2.6   2014-09-08 CRAN (R 3.2.0)                  
    htmlwidgets   0.5     2015-06-21 CRAN (R 3.2.0)                  
    httr          1.0.0   2015-06-25 CRAN (R 3.2.0)                  
    jsonlite      0.9.19  2015-11-28 CRAN (R 3.2.2)                  
    labeling      0.3     2014-08-23 CRAN (R 3.2.2)                  
    magrittr      1.5     2014-11-22 CRAN (R 3.2.2)                  
    MASS          7.3-43  2015-07-16 CRAN (R 3.2.2)                  
    memoise       0.2.1   2014-04-22 CRAN (R 3.2.2)                  
    munsell       0.4.2   2013-07-11 CRAN (R 3.2.2)                  
    plotly      * 2.0.5   2015-12-01 Github (ropensci/plotly@6053d9b)
    plyr          1.8.3   2015-06-12 CRAN (R 3.2.2)                  
    proto         0.3-10  2012-12-22 CRAN (R 3.2.2)                  
    R6            2.1.1   2015-08-19 CRAN (R 3.2.2)                  
    Rcpp          0.12.2  2015-11-15 CRAN (R 3.2.2)                  
    reshape2      1.4.1   2014-12-06 CRAN (R 3.2.0)                  
    scales        0.3.0   2015-08-25 CRAN (R 3.2.0)                  
    stringi       1.0-1   2015-10-22 CRAN (R 3.2.0)                  
    stringr       1.0.0   2015-04-30 CRAN (R 3.2.2)                  
    viridis       0.3.1   2015-10-11 CRAN (R 3.2.0)                  
    yaml          2.1.13  2014-06-12 CRAN (R 3.2.0)

**Comment by tom-christie** [comment]
Ah hah! I installed the CRAN version (2.0.3) and I don't get this behavior.

**Comment by cpsievert** [comment]
Ah, I see it now, here is an even more minimal example:

```r
library(plotly)
config(plot_ly(),showLink=F, sendData=F, displaylogo=F, displayModeBar = F)
```

In the CRAN version we're using plotlyjs 1.0.0, but I recently updated to 1.1.0. Is this a known plotlyjs bug @etpinard?

**Comment by etpinard** [comment]
fixed in `v1.1.1` https://github.com/plotly/plotly.js/releases/tag/v1.1.1
